src/H5Ztrans.c

```diff
@@ -178,8 +177,6 @@ static void       H5Z__xform_reduce_tree(H5Z_node *tree);
             H5Z_XFORM_DO_OP1((RESL), (RESR), float, OP, (SIZE))                                              \
         else if ((TYPE) == H5T_NATIVE_DOUBLE)                                                                \
             H5Z_XFORM_DO_OP1((RESL), (RESR), double, OP, (SIZE))                                             \
-        else if ((TYPE) == H5T_NATIVE_LDOUBLE)                                                               \
```

Comment:
Why remove?

Reply:
Good catch. I removed the wrong half of that big #if. Fixed.